### ISSUE TITLE
Round ends when tickets are active with admins

### ISSUE THREAD

**Issue by Time-Green** [yogstation13/yogstation-classic#662]
### Expected Behaviour

Round delays
### Actual Behaviour

Round is rachist and ignores the existence of moderators
### Steps to reproduce the problem

![image](https://cloud.githubusercontent.com/assets/7501474/13992576/eadbba34-f11c-11e5-8f6f-5e38d412eef5.png)

NO IDEA

**Comment by ShadowDeath6** [comment]
Modmins not real admins. 

Probably a problem with gang mode since it has a weird round-end condition.

**Comment by Time-Green** [comment]
The next round it happened again.
![image](https://cloud.githubusercontent.com/assets/7501474/13996762/6815b024-f12f-11e5-8284-69ac2104f3de.png)

**Comment by Time-Green** [comment]
And again. This needs to get fixed right now because it ruins tickets and happens every round
![image](https://cloud.githubusercontent.com/assets/7501474/13998188/34f77c34-f136-11e5-9e75-87bc184b3261.png)

**Comment by ShadowDeath6** [comment]
Critical tag pls

**Comment by Cruix** [comment]
This was done by Kn0ss0s' update that made it so afk admins didn't hold up the round restarting. In the process, he fixed a long standing bug that caused all admins, not just those with +SERVER, to count for holding up the round end. This may be intentional.

**Comment by Time-Green** [comment]
Ok, something was done.
It will now give the end round prompt at every moment all tickets are closed.
This caused server to restart

**Comment by DavidEdwards** [comment]
Yes, funnily enough, this is how the ticket system was always intended to work. However, after we added the ability for the last ticket to restart the server, it wasn't necessary anymore to require +SERVER. I have added a new permission +TICKET for users that the server will delay for, to allow ticket replies. Other people can still reply to tickets, but the server won't wait for them at the end of a round. Admins, moderators, etc, will cause a delay on the server. However, coders will not.
